README.md

```diff
@@ -812,36 +812,36 @@ jobs:
+Note that the `develocity.gradle.org` server requires authentication in order to publish scans. The provided `DEVELOCITY_ACCESS_KEY` isn't required by the Develocity injection script, 
 but will be used by the GE plugin in order to authenticate with the server.
 
-## Configuring Gradle Enterprise injection
+## Configuring Develocity injection
 
 The `init-script` supports a number of additional configuration parameters that you may fine useful. All configuration options (required and optional) are detailed below:
 
-| Variable | Required | Description |
-| --- | --- | --- |
-| GRADLE_ENTERPRISE_INJECTION_ENABLED | :white_check_mark: | enables Gradle Enterprise injection |
-| GRADLE_ENTERPRISE_URL | :white_check_mark: | the URL of the Gradle Enterprise server |
-| GRADLE_ENTERPRISE_ALLOW_UNTRUSTED_SERVER | | allow communication with an untrusted server; set to _true_ if your Gradle Enterprise instance is using a self-signed certificate |
-| GRADLE_ENTERPRISE_ENFORCE_URL | | enforce the configured Gradle Enterprise URL over a URL configured in the project's build; set to _true_ to enforce publication of build scans to the configured Gradle Enterprise URL |
-| GRADLE_ENTERPRISE_PLUGIN_VERSION | :white_check_mark: | the version of the [Gradle Enterprise Gradle plugin](https://docs.gradle.com/enterprise/gradle-plugin/) to apply |
-| GRADLE_ENTERPRISE_CCUD_PLUGIN_VERSION |  | the version of the [Common Custom User Data Gradle plugin](https://github.com/gradle/common-custom-user-data-gradle-plugin) to apply, if any |
-| GRADLE_ENTERPRISE_PLUGIN_REPOSITORY_URL |  | the URL of the repository to use when resolving the GE and CCUD plugins; the Gradle Plugin Portal is used by default |
+| Variable                          | Required | Description |
+|-----------------------------------| --- | --- |
+| DEVELOCITY_INJECTION_ENABLED      | :white_check_mark: | enables Develocity injection |
+| DEVELOCITY_URL                    | :white_check_mark: | the URL of the Develocity server |
+| DEVELOCITY_ALLOW_UNTRUSTED_SERVER | | allow communication with an untrusted server; set to _true_ if your Develocity instance is using a self-signed certificate |
+| DEVELOCITY_ENFORCE_URL            | | enforce the configured Develocity URL over a URL configured in the project's build; set to _true_ to enforce publication of build scans to the configured Develocity URL |
+| DEVELOCITY_PLUGIN_VERSION         | :white_check_mark: | the version of the [Develocity Gradle plugin](https://docs.gradle.com/enterprise/gradle-plugin/) to apply |
+| DEVELOCITY_CCUD_PLUGIN_VERSION    |  | the version of the [Common Custom User Data Gradle plugin](https://github.com/gradle/common-custom-user-data-gradle-plugin) to apply, if any |
+| DEVELOCITY_PLUGIN_REPOSITORY_URL  |  | the URL of the repository to use when resolving the Develocity and CCUD plugins; the Gradle Plugin Portal is used by default |
```

Comment:
I'd like to rename this to `GRADLE_PLUGIN_REPOSITORY_URL`, since it makes no sense to have different env vars to configure the plugin portal for the DV plugins and the dependency-graph plugin.
I'll take care of this in a subsequent change, if necessary.

Reply:
Agree, changed